core/arch/instr_shared.c

```diff
@@ -297,6 +299,8 @@ instr_build(dcontext_t *dcontext, int opcode, int instr_num_dsts, int instr_num_
 {
     instr_t *instr = instr_create(dcontext);
     instr_set_opcode(instr, opcode);
+    if (instr_is_label(instr))
+        instr_set_label_callback(NULL, instr);
```

Comment:
I don't think this is necessary and it will add overhead: we can rely on the memset in instr_init(), right?

Reply:
Good point. I removed and tested successfully.